Description: Use utf8 encoding when loading tests
 Test files are encoded in utf8, normally Python3 uses locale settings
 and decodes, but this fails when building as locale is not set, and
 could also be set to some unexpected value.
 .
 relational (2.0-1) unstable; urgency=medium
 .
   * New upstream release
   * Now uses Python3
   * Now uses PyQt5
   * Bump standards version
   * Recommends python3-pyqt5.qtwebkit to open the documentation from the UI but
     it is not a necessity
Author: Salvo 'LtWorf' Tomaselli <tiposchi@tiscali.it>

--- relational-2.0.orig/driver.py
+++ relational-2.0/driver.py
@@ -40,7 +40,7 @@ tests_path = 'test/'
 
 def readfile(fname):
     '''Reads a file as string and returns its content'''
-    fd = open(fname)
+    fd = open(fname,encoding='utf-8')
     expr = fd.read()
     fd.close()
     return expr
